xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -644,6 +676,26 @@ bool CPVRChannelGroup::AddAndUpdateChannels(const CPVRChannelGroup &channels, bo
   return bReturn;
 }
 
+bool CPVRChannelGroup::IsMissingChannelsFromClient(int iClientId) const
+{
+  for (int iFailedClientID : m_failedClientsForChannels)
```

Comment:
Could perhaps be shortened to something like `return std::find(bla.begin(), bla.end(), iClientId) != bla.end()`

Reply:
yeah